README.md

```diff
@@ -175,10 +206,24 @@ If connecting to a development server over http, make sure to turn on http suppo
 ### Run in the emulator
 
 ```
-$ npx cordova emulate ios
+npm run <ios>
+```
 AND/OR
-$ npx cordova emulate android
 ```
+npm run <android>
+```
```

Comment:
It is not clear what `<ios>` and `<android>` are.

I think we should just give examples of the most common build scripts that we think will be used and list them out here.

`npm run build` (to build for both Android and iOS platforms)
`npm run build-prod-android` (to build for Android platform only)
`npm run build-prod-ios` (to build for iOS platform only)